packages/dashboard/src/components/Wizard/FullParentWizard.tsx

```diff
@@ -0,0 +1,10 @@
+import { styled } from 'office-ui-fabric-react/lib/Utilities';
+import { FullParentWizardBase } from './FullParentWizard.Base';
+import { IFullParentWizardProps, IFullParentWizardStyleProps, IFullParentWizardStyles } from './FullParentWizard.types';
+import { getFullParentWizardStyles } from './FullParentWizard.styles';
+
+export const FullParentWizard: (props: IFullParentWizardProps) => JSX.Element = styled<

```

Comment:
>FullParentWizard [](start = 13, length = 16)

Do you think the name of this control can be "FullPageWizard"?

Reply:
no, this is actually a FullParentWizard - it takes the size of its parent. It is hard to determine the size of a full page (i.e. where is the the navbar and top bar, etc). The user needs to instantiate this element inside a "container" element that should be the size of a page to create a full page wizard.